packages/specification/schemas/resources/OrganisationPrescribing.yaml

```diff
@@ -1,5 +1,5 @@
 type: object
-description: Details of prescribing organisation (primary care) or the managing organisation (secondary care) for a prescription.
+description: Details of the prescribing organisation (primary care) or the managing organisation (secondary care) for a prescription.
```

Comment:
Just 
"Details of the organisation that a prescription was prescribed within."

Reply:
Changed as suggested.